dawn/src/driver-includes/to_vtk.cpp

```diff
@@ -117,16 +117,17 @@ static std::string formatNaNs(const double value) {
 
 namespace {
 StencilFieldsVtkOutput& getStencilFieldsVtkOutput(int num_k, std::string stencil_name, int iter) {
-  if(stencil_to_output_map.count(std::make_pair(stencil_name, iter - 1))) {
-    stencil_to_output_map.erase(std::make_pair(stencil_name, iter - 1));
-  }
```

Comment:
> Calling the destructor the second time a serialization routine is called is not safe. A single stencil may serialize multiple fields in the same iteration

Hmm, I don't see how it's possible that the destructor of the same object gets called more than once, as once the entry of `stencil_to_output_map` with some key is deleted (which entails 1 call to the destructor), the next time the condition (of the if) is going to be evaluated to false, as the key is not going to be present anymore.
Moreover this code destroys the object for the same stencil but for the previous iteration (`iter - 1`), for which all fields have already been serialized.
I think the problem then it's somewhere else, but ....